### DISCUSSION TITLE
Custom leave message not working on mc 1.21

### DISCUSSION THREAD

**Discussion by globekeeperjb** [EssentialsX/Essentials#5867] [Q&A]
Hello, the custom leave message was always working, but now on mc 1.21, ES version 2.20.1 it is not working (custom join message is working normally) - I know that it is for now unsupported version, can you expect, when the 2.21.0 version will come out? It is possible to fix this issue for now on these versions I am using? Thanks.

**Comment by Evidentsinger14** [comment]
1.21 is supported on latest development builds. Can you explain a little bit more what you mean by custom leave messages aren't working?

**Reply by globekeeperjb** [reply]
When player disconnects, the leave message does not show... I have server on aternos, and it is not possible to upload downloaded plugin, only plugins, which are approved for aternos, and they don't have 2.21.0.

**Reply by globekeeperjb** [reply]
![1](https://github.com/EssentialsX/Essentials/assets/175157178/f0d8d88f-2cc6-4f0c-8a28-8f32d6701aa9)

**Comment by Chew** [comment]
We can't accept bug reports from outdated versions. It's up to your host to provide you with a way to update to the latest development build, or you'll have find a new host.

**Reply by globekeeperjb** [reply]
I have fixed the problem couple hours ago. Thanks for trying to help me, have a nice day.
